misc/local_tests.conf

```diff
@@ -8,7 +8,7 @@ build local/docker
 
 # Add the new test into the list.
 add flaky
-
+add mac
```

Comment:
this should be macoui (specific test name) and not mac.

Reply:
Changed to macoui